symphony/lib/toolkit/class.administrationpage.php

```diff
@@ -1066,11 +1072,15 @@ private function buildSectionNavigation(&$nav)
                  *  The computed page limit for the current page
                  * @param string $page_url
                  *  The computed page url for the current page
+                 * @param int $section.id
+                 *  The id of the section for this url
+                 * @param string $section.handle
+                 *  The handle of the section for this url
                  */
                 Symphony::ExtensionManager()->notifyMembers('CanAccessPage', '/backend/', array(
                     'allowed' => &$hasAccess,
-                    'page_limit' => $c['limit'],
-                    'page_url' => $c['link'],
+                    'page_limit' => 'author',
```

Comment:
I think you got mixed up (like I was) because of the way the diff is displayed. There are a lot of code in between ;)